client/src/components/Signup/actions.js

```diff
@@ -0,0 +1,27 @@
+import axios from 'axios'
+
+export const USER_REGISTRATION_SUCCESSFUL = "USER_REGISTRATION_SUCCESSFUL"
+export const USER_REGISTRATION_FAILED = "USER_REGISTRATION_FAILED"
+
+export const registrationSuccessful = (payload) => {
+    return {
+        type: USER_REGISTRATION_SUCCESSFUL,
+        payload
+    }
+}
+
+export const registrationFailed = (payload) => {
+    return {
+        type: USER_REGISTRATION_FAILED,
+        payload
+    }
+}
+
+// thunk
+export const registerUser = (payload) => {
+    return (dispatch) => {
+        return axios.post('http://localhost:3000/signup', payload)
```

Comment:
Instead of using static urls here, do you consider using environment variables so that when we are deploying later, these kind of bugs can be avoided.

Reply:
Yup, makes perfect sense. Shall update it.